app/uploaders/logo_uploader.rb

```diff
@@ -0,0 +1,49 @@
+class LogoUploader < BaseUploader
+  process :resize_image
+  EXTENSION_ALLOWLIST = %w[svg jpg jpeg png].freeze
+
+  def store_dir
+    "uploads/logos/"
+  end
+
+  def extension_allowlist
+    EXTENSION_ALLOWLIST
+  end
+
+  def image_type_whitelist
+    %i[svg jpg jpeg png]
+  end
+
+  def size_range
+    # TODO: decide with @nickytonline on a size range that
+    # matches the frotend validation.
+    1..(3.megabytes)
+  end
+
+  def resize_image
```

Comment:
@nickytonline I added a resize method, we can decide on the calculation and amend as necessary.